### DISCUSSION TITLE
Can this help us create an adapter for XG Mobile interface to Oculink interface?

### DISCUSSION THREAD

**Discussion by niraj97** [osy/XG_Mobile_Station#10] [General]
Since we now have the detail pins information of XG mobile. can this help us to create an adapter for XG Mobile interface to Oculink interface?
Sorry if i sound dumb. I dont have any knowledge about PCB designs.

**Comment by osy** [comment]
Yeah you can build one just replace the PCIe slot with an oculink port, add some redrivers, and route it all up.

**Reply by digitalsignalperson** [reply]
Is it possible to make a XGM port to Oculink adapter? For instance: have an off the shelf oculink eGPU dock, and you could connect the cable to an ASUS laptop with an adapter that maps the oculink connector to the XGM port.

**Reply by osy** [reply]
Is it possible? Yes
Is anyone doing it? Not that I'm aware of

The info from this repository is enough for someone to work on this.
